user-query-optimizer/athena_optimizer.py

```diff
@@ -0,0 +1,23 @@
+import sys
+sys.path.append('../user-query-optimizer/optimizations')
+import approximates
+import column_selection
+import partitions
+import nested_subqueries
+
+class AthenaOptimizer:
```

Comment:
This is not exactly how we want the class structure to work. What we want is for e.g. `AthenaOptimizer` to inherit from `Optimizer`. For that to happen the class needs to be declared as `class AthenaOptimizer(Optimizer)`, and then the `Optimizer` class needs to return an instance of an `AthenaOptimizer` if the database string is "Athena". Ping me if you have questions about this.